src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -615,3 +615,122 @@ app.use(function processItems(req, res, next) {
   })
 });
 ```
+
+### Vue.js
+
+Vue Tracing Integration allows us to track rendering performance during initial application load.
+
+It is achieved by injecting our handler inside Vue's `beforeCreate` mixin, which gives us an access to component during it is all lifecycle stages.
+When we encounter component named `root`, which is top-level Vue instance (as in `new Vue({})`), we use our APM Tracing integration,
+and create a new activity named `Vue Application Render`. Once it is created, it will wait until all it is child components are rendered, and there are no new rendering events triggered within configured `timeout`, before marking the activity as completed.
```

Comment:
```suggestion
and create a new activity named `Vue Application Render`. Once it is created, it will wait until all of it's child components are rendered, and there are no new rendering events triggered within configured `timeout`, before marking the activity as completed.
```